_includes/pbc/all/install-features/202410.0/install-the-product-management-ai-module.md

```diff
@@ -0,0 +1,195 @@
+
+This document describes how to install the ProductManagementAi module.
+
+## Install feature core
+
+Follow the steps below to install the ProductManagementAi module core.
+
+### Prerequisites
+
+Install the required features:
+
+| NAME                | VERSION          | INSTALLATION GUIDE                                                                                                                                                                                   |
+|---------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OpenAi              | {{page.version}} | [Install the OpenAi feature](/docs/pbc/all/miscellaneous/{{page.version}}/third-party-integrations/open-ai/install-and-upgrade/install-the-open-ai-module.html)                                      |
+| Product             | {{page.version}} | [Install the Product feature](/docs/pbc/all/product-information-management/{{page.version}}/base-shop/install-and-upgrade/install-features/install-the-product-feature.html)                         |
+| Category Management | {{page.version}} | [Install the Category Management feature](/docs/pbc/all/product-information-management/{{page.version}}/base-shop/install-and-upgrade/install-features/install-the-category-management-feature.html) |
+
+### 1) Install the required modules
+
+Run the following command to install the required module:
+
+```bash
+composer require spryker-eco/product-management-ai:"^0.1.1" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following modules have been installed:
+
+| MODULE              | EXPECTED DIRECTORY                       |
+|---------------------|------------------------------------------|
+| ProductManagementAi | vendor/spryker-eco/product-management-ai |
+
+{% endinfo_block %}
+
+### 2) Set up database schema and transfer objects
```

Comment:
For projects that don't have  `ProductMerchantPortalGui` module we still need to extend ProductAbstractTransfer:

```
<transfer name="ProductAbstract">
    <property name="categoryIds" type="int[]" singular="idCategory"/>
</transfer>

Reply:
It is in module definition already: [install-the-product-management-ai-module.md](https://github.com/spryker/spryker-docs/pull/2905/files/8b06c3b7649a69f7896806b906a9847b6233d61b#diff-91b842efdcd2728293c61b16d105ffcba80342a0d052a8af42187c73938e4272)